packages/games/tools/retrorun/retrorun.sh

```diff
@@ -104,6 +116,18 @@ else
 fi
 
 
+# Force left analog stick to DPAD
+# Get configuration from distribution.conf and set to FORCE_ANALOG_STICK
+get_setting "force_left_analog_stick"
+echo ${EES}
+if [ "${EES}" == "auto" ] || [ "${EES}" == "false" ] || [ "${EES}" == "none" ] || [ "${EES}" == "0" ]; then
+	sed -i "/^retrorun_force_left_analog_stick/d" ${RRCONF}
+	echo 'retrorun_force_left_analog_stick = auto' >> ${RRCONF}
```

Comment:
this doesn't seem to be right it should be true or false depending on what is the default setting

Reply:
Should be ok now, but I see the beta stuck displaying our logo at startup...